<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainClass</class>
 <widget class="QWidget" name="mainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">main</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">Text1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string notr="true">Text2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string notr="true">Text3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
